test/benchmarks/driver_bench/src/main.mts

```diff
@@ -198,6 +199,27 @@ function calculateCompositeBenchmarks(results: MetricInfo[]) {
   return [...results, ...compositeResults];
 }
 
-const finalResults = calculateCompositeBenchmarks(results);
+function calculateNormalizedResults(results: MetricInfo[]): MetricInfo[] {
+  const primesBench = results.find(r => r.info.test_name === 'primes');
+  const pingBench = results.find(r => r.info.test_name === 'ping');
 
-await fs.writeFile('results.json', JSON.stringify(finalResults, undefined, 2), 'utf8');
+  assert.ok(pingBench, 'ping bench results not found!');
+  assert.ok(primesBench, 'primes bench results not found!');
+  const pingThroughput = pingBench.metrics[0].value;
+  const primesThroughput = primesBench.metrics[0].value;
+  primesBench.metrics.push({ 'name': 'normalized_throughput', value: NORMALIZED_PING_SCALING_CONST * pingThroughput / primesThroughput });
```

Comment:
this should be just primesThroughput, no fancy math, taking my other comment into account